app/views/section1/link_ducr_to_mucr.scala.html

```diff
@@ -75,7 +75,7 @@
             ))
         )
 
-        @tariffExpander(LinkDucrToMucr, request.declarationType, Some("tariff.declaration.linkDucrToMucr.title"))(messages, appConfig)
```

Comment:
The injection of `tariffExpander` and `AppConfig`, as well as the `@import`, should be removed